packages/shared/src/graphql/search.ts

```diff
@@ -72,6 +72,7 @@ export const SEARCH_POST_SUGGESTIONS = gql`
 export const SEARCH_POST_RECOMMENDATION = gql`
   query SearchQuestionRecommendations {
     searchQuestionRecommendations {
+      id
```

Comment:
With this addition we actually have to be very careful.
It will push these as we already have a session and it won't exist yet. 😅

So maybe we have to escape it somewhere. to prevent urls from being wrong.

Reply:
@rebelchris good point, I updated so the URL always includes just `q` query param for suggestions -- is there any reason we would ever want to pass the ID here?